lib/ModelMixins/CatalogMemberMixin.ts

```diff
@@ -78,6 +78,17 @@ function CatalogMemberMixin<T extends Constructor<CatalogMember>>(Base: T) {
       );
     }
 
+    @computed
+    get infoAsObject() {
```

Comment:
This supports accessing infoSections within the mustache templates for feature info